si.tano.Vremenar.yaml

```diff
@@ -0,0 +1,59 @@
+---
+app-id: si.tano.Vremenar
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vremenar
+rename-icon: vremenar
+rename-desktop-file: vremenar.desktop
+copy-icon: true
+finish-args:
+  # Required due to being a GUI application
+  - "--socket=x11"
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - "--share=ipc"
+  # Required for experimental wayland support
+  - "--socket=wayland"
```

Comment:
...but if this is experimental, it's possibly better to leave it out.

Reply:
Honestly I have yet to test this. Will try to do this today.

Reply:
It seems it does not work after all. Will investigate this in the future, but probably I need to rebuild the app so at the moment it will be x11-only.